lib/SieveKeywordRegistry.php

```diff
@@ -187,38 +181,24 @@ public function isTest(string $name): bool
 
     /**
      * Is command.
-     *
-     * @param string $name
-     * @return bool
      */
     public function isCommand(string $name): bool
     {
         return isset($this->commands[$name]);
     }
 
-    /**
-     * @param string $name
-     * @return mixed|null
-     */
-    public function matchType($name)
+    public function matchType(string $name): mixed
     {
         return $this->matchTypes[$name] ?? null;
     }
 
-    /**
-     * @param string $name
-     * @return mixed|null
-     */
     public function addressPart(string $name)
```

Comment:
```suggestion
    public function addressPart(string $name): mixed
```
to have everything typed (same for the others)